spec/features/tasks/new_spec.rb

```diff
@@ -0,0 +1,31 @@
+require 'rails_helper'
+
+RSpec.describe 'Create a task', type: :feature do
+  include_context 'when user signed in'
+
+  before do
+    visit projects_path
+    click_on 'New Project'
+
+    fill_in 'Name', with: 'A test project'
+    fill_in 'Description', with: 'Test description'
+
+    click_button 'Create Project'
+  end
+
+  scenario 'user creates a task' do
+    visit tasks_path
+    click_on 'New Task'
+
+    select "A test project", :from => "Select a project"
+    fill_in 'Title', with: 'A test task'
+    fill_in 'Description', with: 'Test description'
+    fill_in 'Deadline', with: 'November 27, 2021 23:59'
+
+    click_button 'Save'
+
+    expect(page).to have_content("Task has been created!")
```

Comment:
It is also a good idea to check what is on the page after creating the task. It is not necessary, just FYI
```suggestion
    expect(page).to have_content("Task has been created!")
    
    expect(page).to have_content("A test task")
    expect(page).to have_content("Test description")
```